lib/player.js

```diff
@@ -2701,7 +2729,17 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         this.manifest_,
         this.config_.streaming,
         startTime,
-        () => this.onSeek_());
+        () => this.onSeek_(),
+        (event) => {
+          if (event.type === shaka.util.FakeEvent.EventName.StallDetected) {
```

Comment:
I think this is now unnecessary.  You can just dispatch any event sent through this callback:

```js
  (event) => this.dispatchEvent(event),
```